geode-wan/src/main/java/org/apache/geode/cache/client/internal/locator/wan/WanLocatorDiscovererImpl.java

```diff
@@ -65,11 +66,19 @@ private void exchangeLocalLocators(int port, DistributionConfigImpl config,
       LocatorMembershipListener locatorListener, final String hostnameForClients) {
     String localLocator = config.getStartLocator();
     DistributionLocatorId locatorId = null;
+
+    InternalDistributedSystem system = InternalDistributedSystem.getConnectedInstance();
```

Comment:
same comment - we want to stop using this static method to find an InternalDistributedSystem, and you have a DistributionConfig parameter that provides the memberName that you need.